README.md

```diff
@@ -50,220 +18,15 @@ Moving to the [golang GitHub organization](https://github.com/golang) brings us
-* `Go: Run on Go Playground` Upload the current selection or file to the Go Playground
-
-You can access all of the above commands from the command palette (`Cmd+Shift+P` or `Ctrl+Shift+P`).
-
-A few of these are available in the editor context menu as an experimental feature as well. To control which of these commands show up in the editor context menu, update the setting `go.editorContextMenuCommands`.
-
-
-### _Optional_: Debugging
-
-To use the debugger, you must currently manually [install delve](https://github.com/derekparker/delve/tree/master/Documentation/installation). For more read [Debugging Go Code Using VS Code](https://github.com/microsoft/vscode-go/blob/master/docs/Debugging-Go-code-using-VS-Code.md).
-
-#### Debugging in WSL
-
-If using WSL on Windows, you will need the WSL 2 Linux kernel.  See [WSL 2 Installation](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) and note the Window 10 build version requirements. 
-
-#### Remote Debugging
-
-To remote debug using VS Code, read [Remote Debugging](https://github.com/microsoft/vscode-go/blob/master/docs/Debugging-Go-code-using-VS-Code.md#remote-debugging).
-
-## Install or update all dependencies
-
-To quickly get all dependencies installed (or updated) see the [Go Tools wiki page](https://github.com/microsoft/vscode-go/blob/master/docs/Go-tools-that-the-Go-extension-depends-on.md).
-
-## Building and Debugging the Extension
-
-You can set up a development environment for debugging the extension during extension development.
-Read more at [Building, Debugging and Sideloading the extension in Visual Studio Code](https://github.com/microsoft/vscode-go/blob/master/docs/Building,-Debugging-and-Sideloading-the-extension-in-Visual-Studio-Code.md).
-
-## Tools this extension depends on
-
-This extension uses a host of Go tools to provide the various rich features. These tools are installed in your GOPATH by default. If you wish to have the extension use a separate GOPATH for its tools, provide the desired location in the setting `go.toolsGopath`. Read more about this and the tools at [Go tools that the Go extension depends on](https://github.com/microsoft/vscode-go/blob/master/docs/Go-tools-that-the-Go-extension-depends-on.md).
-
-## Frequently Asked Questions (FAQ)
-
-Please see our wiki on [Frequently Asked Questions](https://github.com/microsoft/vscode-go/blob/master/docs/Go-with-VS-Code-FAQ-and-Troubleshooting.md) to get answers to your questions or get started with troubleshooting.
-
 ## Contributing
 
-This project welcomes contributions and suggestions. Please go through our [Contributing Guide](https://github.com/Microsoft/vscode-go/blob/master/CONTRIBUTING.md)
-to learn how you can contribute. It also includes details on the Contributor License Agreement.
+This project welcomes contributions and suggestions. Please go through the [Contributing Guide](https://github.com/golang/vscode-go/blob/master/CONTRIBUTING.md) in the new repository.
```

Comment:
@hyangah, @stamblerre Are you planning to keep the Contributing guide in the new repo or does Google projects follow a different format?

Reply:
Wow, I did all that documentation work and didn't even notice we had that page in there still. I will delete it, we have a totally different process, documented here: https://github.com/golang/vscode-go/blob/master/docs/contributing.md.

Reply:
I think we could probably just delete the Contributing and Code of Conduct sections, as they aren't really necessary anymore and the source of truth is in the new repository.

Reply:
I'd prefer to keep the two sections. For people who just take a glance at this, I still want to communicate that we welcome contributions.

Pushed an update to update the link. Please take another look

Reply:
My preference is to keep this README as simple as possible, so that it's clear to readers that they are looking in the wrong place. I worry that welcoming contributions while directing people to a different repository might send a mixed message.

In any case, archiving the repository will make a pretty clear statement, so either way is fine with me.